_posts/commands/2020-05-01-mom_spectate.md

```diff
@@ -7,4 +7,4 @@ tags:
   - lobby
 ---
 
-Starts spectating. Starts in free roam spectate mode if there are no active ghosts (players) on your map.
\ No newline at end of file
+Allows you to start spectating. Starts in free roam spectate mode if there are no active ghosts (players) on your map.
```

Comment:
This doesn't allow you to start spectating, it starts spectating; the wording is good already.